src/backends/unix/system.c

```diff
@@ -487,6 +487,25 @@ Sys_Remove(const char *path)
 	remove(path);
 }
 
+void
+Sys_RemoveDir(const char *path)
+{
+	DIR *directory = opendir(path);
+	struct dirent *file;
+
+	if (Sys_IsDir(path))
+	{
+		while ((file = readdir(directory)) != NULL)
+		{
+			Sys_Remove(va("%s%s", path, file->d_name));
```

Comment:
va() is not good here - we support paths with up to 4096 chars (`MAX_OSPATH`), but va() uses a 1024 char buffer.